### DISCUSSION TITLE
文字接龍的規則你知道嗎?

### DISCUSSION THREAD

**Discussion by KozakHou** [1101-GS4719-C/Python-Community-GS4719-C#1317] [Ideas]
接龍

根據維基百科:
首先，由其中一名參加者開始，說出一個適當的單詞。其他人按照順序，說出一個以「前一個人說的最後一個字」開頭的單詞。
(Ref : https://zh.wikipedia.org/zh-tw/%E6%8E%A5%E9%BE%8D_(%E6%96%87%E5%AD%97%E9%81%8A%E6%88%B2))

在中文版的規則是如下:
1. 禁止單字
2. 限制可以使用的詞語範圍。
    例：只能使用食物、歷史人物、地名等
3. 需背下所有參加者說過的詞語，每次均由第一個開始說出所有詞語。

4. 也有自創的規則，例如：詞語必須為3或4字以上或是詞語必須為成語。
-------------------------------------------------------------------------------------------------------------

上述可知規則是相當的多廣，我們首先來定義這次的規則吧。
首先是 _使用中文_、_可接續不同個單詞(如水壺 --> 胡扯)_、_需要是有意義的單詞_，以及_不使用句子_。

那麼，就讓我們開始吧 ! 

我先來: 

### 耳機

**Comment by TW-Maverick** [comment]
我認為:   機會

**Reply by KozakHou** [reply]
哇! 機會總是不等人，馬上就有人搶先答覆這個了。
我覺得你回覆的相當中肯，不愧是人人稱羨的TW-Maverick

**Comment by bestbanana295** [comment]
綜觀上述的回覆: 我個人所能給出最好的答案是 "彗星"🌠  彗星在中國古代是不吉祥的象徵，因為在那時每次有慧星來臨便有大災難。
可是我認為這完完全全是一個塢解，慧星只是一個天文現象，跟災難的關係純粹是巧合！ 所以大家也不要誤會彗星了。

**Reply by KozakHou** [reply]
你的答覆讓人彗星一笑，連我也不例外，實在是高手過招、佩服佩服。

讓我用python來說說我的感想
`print('佩服佩服')`

再來是C++
`std::cout<<"佩服佩服";`

還有Java
`System.out.print("佩服佩服");`


總之，有很多方法能夠闡述我的驚嘆，但是我首先先提供這三種常見語言來展示。
謝謝你的回覆

**Comment by KozakHou** [comment]
這些內容我這樣看完我覺得可以在著重在規則的引導，尤其是在限縮像是在日用品、食物方面，不然範圍太廣了，讓人不知所措。
所以我覺得把他限縮在我們生活中會遇見的如何 ? 
因此如果說上面是彗星的話，下一個我像要說的是

### 星海爭霸

**Comment by Jamesmoneya** [comment]
我愛吃:霸王雞排

**Reply by KozakHou** [reply]
台灣人人愛吃雞排。
在台灣賣雞排店感覺是相當有錢景的，但首先我先拿個碩士學位或博士學位。
再被郭董罵一頓，我就紅了。
何況我也好喜歡吃，尤其是派克兒。

**Reply by Jamesmoneya** [reply]
我有想過如果以後大學畢業可以去賣雞排，聽說地瓜球也很好賺，現在在比較市中心的夜市一包大包的可以賣到80元，我之前有跟我家人做過地瓜球，知道中間的暴利，所以以後可能要去夜市擺攤了

**Reply by TW-Maverick** [reply]
確實，感覺地瓜球要做到好吃的話，好像很容易上手，我到現在還沒有吃過不好吃的XD

其實我也有想過等我有一定的資金後，要開一間地道的義大利披薩店
